rita/src/exit.rs

```diff
@@ -223,6 +223,7 @@ fn main() {
             .route("/settings", Method::GET, get_settings)
             .route("/settings", Method::POST, set_settings)
             .route("/version", Method::GET, version)
+            .route("/wipe", Method::POST, wipe)
```

Comment:
Same here. BTW did you test that this isn't visible to machines on the outside? Imagine someone developing on `borked` and suddenly someone starts messing with them by wiping the exit.

Reply:
that's currently a firewall thing the installer should *not* open the port these endpoints reside on. Now that's not the best way to handle things long term but it works for now. 

In general unauthenticaed endpoints are an unavoidable problem on clients, but on exits they should probably filter for localhost connections only or force auth.

Reply:
Its on the same address as dashboards: `[::0]:$PORT` - isn't `::0` all addresses so probably all dashboard endpoints are possibly visible outside?

Reply:
we format in the port, but we bind to all addresses but block the port with the firewall (or should, I'll check today)

Reply:
should we just bind to 192.168.10.1? or whatever the configured lan is? but that's out of scope for this pull.